launch/tier4_perception_launch/launch/perception.launch.xml

```diff
@@ -35,6 +35,7 @@
   />
   <arg name="use_pointcloud_container" default="false" description="launch pointcloud container"/>
   <arg name="pointcloud_container_name" default="pointcloud_container"/>
+  <arg name="enable_fine_detection" default="true"/>
```

Comment:
How about this?
```suggestion
    <!-- Traffic Light Recognition Parameters -->
    <arg name="traffic_light_recognition/enable_fine_detection" default="true"/>
```

Reply:
Seems nice! 
I commited it by 
https://github.com/autowarefoundation/autoware.universe/pull/1991/commits/c80381231c5ab56f363571bb9343b57fce6d702e.